# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fgetty/makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fgetty-0.6/Makefile	2002-01-24 02:35:22.000000000 +0100
+++ fgetty-0.6-cross/Makefile	2005-02-08 09:14:21.264376496 +0100
@@ -5,18 +5,20 @@
 all: fgetty login login2 checkpassword
 
 DIET=diet
+CC=gcc
+STRIP=strip
 #CROSS=arm-linux-
 CROSS=
 LDFLAGS=-s
 
 %.o: %.c
 #	gcc -march=i386 -mcpu=i386 -pipe -Os -fomit-frame-pointer -I../dietlibc/include -c $^ -DTEST
-	$(DIET) $(CROSS)gcc -pipe -Os -fomit-frame-pointer -I../dietlibc/include -c $^ -DTEST
+	$(DIET) $(CROSS)$(CC) -pipe -Os -fomit-frame-pointer -I../dietlibc/include -c $^ -DTEST
 #	gcc -march=i386 -mcpu=i386 -pipe -g -I../dietlibc/include -DTEST -c $^
-	strip -x -R .comment -R .note $@
+	$(CROSS)$(STRIP) -x -R .comment -R .note $@
 
 %: %.o
-	$(DIET) $(CROSS)gcc -nostdlib -o $@ $^ $(LDFLAGS)
+	$(DIET) $(CROSS)$(CC) -nostdlib -o $@ $^ $(LDFLAGS)
 
 fgetty: fgetty.o fmt_ulong.o
 
@@ -25,7 +26,7 @@
 checkpassword: checkpassword.o
 
 debug: fgetty.c fmt_ulong.o
-	gcc -g -o debug fgetty.c fmt_ulong.o -DDEBUG
+	$(CROSS)$(CC) -g -o debug fgetty.c fmt_ulong.o -DDEBUG
 
 install:
 	install -d $(DESTDIR)/bin $(DESTDIR)/sbin
